src/item.cpp

```diff
@@ -1493,6 +1511,8 @@ std::string item::info(bool showtext, std::vector<iteminfo> &dump_ref) const
                 for( auto &elem : contents ) {
                     const auto mod = elem.type->gunmod.get();
                     temp1.str("");
+                    if( elem.has_flag("IRREMOVABLE") )
+                        temp1 << "[Integrated]";
```

Comment:
Strings that are shown to the user need to be translated, therefor they must be enclosed in `_()`, like this:
```C++
temp1 << _("[...]");
```

The awesome translation team does the translation, that does need to concern the coder.